shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-opengauss/src/main/java/org/apache/shardingsphere/proxy/frontend/opengauss/command/query/extended/bind/OpenGaussComBatchBindExecutor.java

```diff
@@ -46,6 +46,7 @@ public final class OpenGaussComBatchBindExecutor implements CommandExecutor {
     
     @Override
     public Collection<DatabasePacket<?>> execute() throws SQLException {
+        connectionSession.getBackendConnection().handleAutoCommit();
```

Comment:
`PostgreSQLAggregatedBatchedStatementsCommandExecutor` also uses `PostgreSQLBatchedStatementsExecutor` to do batched statements.